frontend/src/pages/Admin/Components/Dashboard/Dashboard.jsx

```diff
@@ -42,22 +59,22 @@ export const Dashboard = (props) => {
     <React.Fragment>
       <h1 className={style["head"]}>Dashboard</h1>
       <div className={style["dash-board"]}>
-        {data.map((d) => (
-          <div
-            className={style["crd"]}
-            key={d.name}
-            onClick={() => props.setTab(d.tab)}
-          >
+        {data.map(d => (
+          <div className={style["crd"]} key={d.name} onClick={() => props.setTab(d.tab)}>
             <div className={style["head1"]}>
               {d.icon}
               <h4> {d.name} </h4>
             </div>
-            <div className={style["content"]}>
-              Get all your {d.name} related details here!
-            </div>
+            <div className={style["content"]}>Get all your {d.name} related details here!</div>
           </div>
         ))}
       </div>
+      <SimpleToast
```

Comment:
Hey @ashuvssut adding it here, it will show the toast message every-time we visit the `dashboard` instead, we need it only when the user clicks on `submit` button of the login form and gets successfully logged in

Reply:
@Kajol-Kumari , done!